packages/dataviews/src/types.ts

```diff
@@ -47,7 +47,7 @@ export type Operator =
 	| 'isAll'
 	| 'isNotAll';
 
-export type FieldType = 'text' | 'integer' | 'datetime' | 'media';
+export type FieldType = 'text' | 'integer' | 'datetime' | 'media' | 'boolean';
```

Comment:
I've noticed we've introduced a `media` fieldtype that doesn't have a corresponding field type definition. I'd like to fix this in a follow-up.